Dockerfile.konflux

```diff
@@ -2,8 +2,7 @@
 ARG SOURCE_CODE=.
 
 # BEGIN -- workaround lack of go-toolset for golang 1.23
-ARG GOLANG_IMAGE=registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder@sha256:b848cf648a383b6e207ebd150afc7e65a90efe0d7734f4f6d2767144d58d68d0
-FROM ${GOLANG_IMAGE} AS golang
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23@sha256:ca0c771ecd4f606986253f747e2773fe2960a6b5e8e7a52f6a4797b173ac7f56 AS golang
```

Comment:
@MohammadiIram I guess the bot to update the digests only works on main, and not the release branches? If it ever does work on the release branches, then I think the tag here should be changed to `rhel_8_golang_1.23`, since it looks like the `v1.23` tag has quietly been updated to rhel9 after the digest that you reference here.

See also: #227